src/renderer/store/helpers.ts

```diff
@@ -22,6 +22,7 @@ const baseMockProject: Omit<Project, 'name'> = {
   filePath: 'fakepath',
   fileExtension: 'mp4',
   transcription: null,
+  savePath: '',
```

Comment:
what is savePath referring to? There was a `savePath` we had before that referred to the export path, its name has been changed to `exportFilePath` I believe.

Reply:
We had hacked around with helpers to make testing easier - I reverted back to an older version before raising the PR. This whole file can just be overwritten with whatever is on develop.